### DISCUSSION TITLE
askpdf等tag失效，无法提取PDF信息

### DISCUSSION THREAD

**Discussion by Dradish0512** [MuiseDestiny/zotero-gpt#254] [Q&A]
![image](https://github.com/MuiseDestiny/zotero-gpt/assets/145782131/999dcbc1-9b47-42a5-8707-34c73b669450)
![dc657220908565f79613d6bb26e93fa](https://github.com/MuiseDestiny/zotero-gpt/assets/145782131/c87733c0-1c16-4c7c-bc7b-1a5dc09ec343)

**Comment by Dradish0512** [comment]
之所以有两张图片是因为，我修改了“askpdf”的命令信息，第一张图片是0.2.5版本的命令信息，第二张图片是0.3.2版本的命令

**Reply by Dradish0512** [reply]
![image](https://github.com/MuiseDestiny/zotero-gpt/assets/145782131/b4eedb93-5375-4238-9d75-dc0594e2c2bb)

**Comment by Jasoncrazy-stu** [comment]
一样的问题，一直用不了askpdf。
根据提供的上下文信息，无法获取到准确的查询问题，请提供更多问题的细节或者明确的查询内容，我将尽力提供相应的帮助和回答。

**Comment by zfb132** [comment]
查看是否是下面的问题
https://github.com/MuiseDestiny/zotero-gpt/issues/256#issuecomment-1788439231
